ray-on-gke/user/kuberay-tpu-webhook/deployments/headless-svc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: headless-svc
```

Comment:
Can we give this service a more meaningful name?

Reply:
Done.